integration_tests/e2e/page-not-found.cy.ts

```diff
@@ -27,7 +29,7 @@ context('404 Page Not Found', () => {
     Page.verifyOnPage(AuthSignInPage)
   })
 
-  it('should redirect to 404 page given user does not have any authorities', () => {
+  it('should redirect to Authorisation Error page given user does not have any authorities', () => {
```

Comment:
Previously this test was going to the 404 page not because the user had no/incorrect roles, but because we had not setup the stub for getting the prisoner by ID !!!, so it was actually that (getting a prisoner without setting up the stub) that was throwing the 404! 🙄